src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -66,10 +67,25 @@ public static Phone parsePhone(String phone) throws ParseException {
     }
 
     /**
-     * Parses a {@code String address} into an {@code Address}.
+     * Parses a {@code String birthdayMonth} into an {@code BirthdayMonth}.
      * Leading and trailing whitespaces will be trimmed.
      *
-     * @throws ParseException if the given {@code address} is invalid.
+     * @throws ParseException if the given {@code birthdayMonth} is invalid.
+     */
+    public static BirthdayMonth parseBirthdayMonth(String birthdayMonth) throws ParseException {
+        requireNonNull(birthdayMonth);
+        String trimmedBirthdayMonth = birthdayMonth.trim();
+        if (!BirthdayMonth.isValidBirthdayMonth(trimmedBirthdayMonth)) {
+            throw new ParseException(BirthdayMonth.MESSAGE_CONSTRAINTS);
+        }
+        return new BirthdayMonth(trimmedBirthdayMonth);
```

Comment:
I think the current implementation is very good. Perhaps if time permits, could consider converting integers 1 - 12 to the respective months Jan-Dec to be displayed as an enhancement 😄

Reply:
Was thinking about this as well, but am not sure whether it'll be too packed to display birthday month in the UI together with the rest of the details. Will discuss with the team!